portfolio/src/main/webapp/style.css

```diff
@@ -58,12 +58,6 @@ body {
   text-decoration: none;
 }
 
-.image-grid-row {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 0 4px;
-}
-
```

Comment:
I don't think you want this removed?

Reply:
this change was in response to a comment left by Andrew on the now closed PR; this was dead css